ln1_f4.py

```diff
@@ -8,74 +8,171 @@
 import subprocess
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+from email.mime.base import MIMEBase
+from email import Encoders
+from openpyxl import Workbook
+from openpyxl.styles import colors, Font, Border, Side
 
+database_file_name = "/home/lax/PycharmProjects/learn1/database4"
+add_database_file_name = "/home/lax/PycharmProjects/learn1/add_database4"
+excel_table_name = "/home/lax/PycharmProjects/learn1/new method test.xlsx"
 
-def get_sysresinfo(command):  # функція виконує консольну команду та повертає результат
+
+def get_sysresinfo(command):  # Executes console command and returns data as list
     command_data = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
     (command_data, err) = command_data.communicate()
-    command_data = command_data.replace(",", ".")  # Заміна ком на крапки для коректного виконання float()
-    return command_data.split()
+    return command_data.replace(",", ".").split()
 
 
-raw = get_sysresinfo("mpstat")  # обробка даних по завантаженості процесора
-cpu_user = float(raw[21])
-cpu_sys = 0
-for x in raw[22:29]:
-    cpu_sys += float(x)
-cpu_total = cpu_user + cpu_sys
-cpu_idle = float(raw[30])
+def get_cpu_info():  # return list format - cpu_user, cpu_sys, cpu_total, cpu_idle
+    cpu_usage_info = []
+    cpu_console_data = get_sysresinfo("mpstat")
+    cpu_usage_info.append(float(cpu_console_data[21]))
+    cpu_sys = 0
+    for x in cpu_console_data[22:29]:
+        cpu_sys += float(x)
+    cpu_usage_info.append(cpu_sys)
+    cpu_usage_info.append(cpu_usage_info[0] + cpu_usage_info[1])
+    cpu_usage_info.append(float(cpu_console_data[30]))
+    return cpu_usage_info
```

Comment:
Instead of using a lot of `append` list methods you could just return list:
`return [float(cpu_console_data[21]), cpu_sys, cpu_usage_info[0] + cpu_usage_info[1], float(cpu_console_data[30])]`
But I don't really like this. because it's not readable for me. Much better workflow:
```
cpu_info_part_1 = float(cpu_console_data[21])
cpu_info_part_2 = cpu_usage_info[0] + cpu_usage_info[1]
cpu_info_part_3 = float(cpu_console_data[30])
...
return [...]
```
But you should change names like `cpu_info_part_1` to readable, like `cpu_sys`, because I don't know what you get from here cpu_console_data[21], etc.